File: cpp/Kode program/MaterialInfo.cpp
#pragma once
#include <iostream>
#include <string>
using namespace std;

class MaterialInfo {
private:
    string type;
    string qualityGrade;
public:
    // Setter dan getter informasi material
    void setType(string t) { type = t; }
    string getType() { return type; }

    void setQualityGrade(string q) { qualityGrade = q; }
    string getQualityGrade() { return qualityGrade; }

    // Menampilkan detail material
    void printMaterial() {
        cout << "--- Material Info ---" << endl;
        cout << "Type: " << type << endl;
        cout << "Quality Grade: " << qualityGrade << endl;
    }
};

File: java/Kode program/MaterialInfo.java
public class MaterialInfo {
    private String type;
    private String qualityGrade;

    public void setType(String t) { type = t; }
    public String getType() { return type; }

    public void setQualityGrade(String q) { qualityGrade = q; }
    public String getQualityGrade() { return qualityGrade; }

    public void printMaterial() {
        System.out.println("--- Material Info ---");
        System.out.println("Type: " + type);
        System.out.println("Quality Grade: " + qualityGrade);
    }
}
